src/types/ProjectType.ts

```diff
@@ -1,3 +1,9 @@
+import { SpriteSheet } from "src/types/SpriteSheetType";
 import { ProjectResponseDto } from "../api";
+import { Map } from "./MapType";
 
-export type Project = ProjectResponseDto
+export type Project =  {
+  spriteSheet: SpriteSheet;
+  map: Map;
+  palette: Uint8Array
+} & ProjectResponseDto;
```

Comment:
I would prefer to have `ProjectResponseDto` be a child rather than a union with your data types. What if a new member conflicts with the ones defined with `Project`?